6.9/windows/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM microsoft/windowsservercore
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
+
+ENV NPM_CONFIG_LOGLEVEL info
+ENV NODE_VERSION 6.9.2
+
+RUN $url = ('https://nodejs.org/dist/v{0}/node-v{0}-x64.msi' -f $env:NODE_VERSION); \
+	Write-Host ('Downloading {0} ...' -f $url); \
+	(New-Object System.Net.WebClient).DownloadFile($url, 'node.msi'); \
```

Comment:
I have [investigated using MSI packages](https://github.com/StefanScherer/dockerfiles-windows/issues/1) in Docker images and came to the conclusion to avoid them if it's possible to reduce the size of the produced Docker image layers.
MSI packages are good for VM's and baremetal machines to install and later uninstall them again. But in Docker images you won't rollback and installation. Instead a different Docker image layer will be created with that newer or older version of the software.
There is a cached version of the MSI package at `C:\Windows\Installer\` and some other binary files will be updated installing such a package.